### PR TITLE
🐛 Handle GitHub Validation Failure if User Exists

### PR DESCRIPTION
**Pull request by jasonBirchall**
## 👀 Purpose

- This PR connects to https://github.com/ministryofjustice/operations-engineering/issues/4507 and relates to a 500 internal server error that appears when a user tries to join an organisation they are already a part of.

## ♻️ What's changed

- Introduced a new error handling method when a user tries to join an organisation they're already part of. Instead of returning an abort statement with a 500 status code, the update now renders a custom page (`already-a-member.html`) informing the user that they are already a member in a more user-friendly way.
- Boolean environment variables in the docker-compose.yaml have been updated from `true/false` to "true"/"false". This change ensures compatibility with systems that require string-type environment variables.

### CODE REVIEW

File: app/templates/pages/already-a-member.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+{{ govukBackLink ({
+'text': 'Back',
+'href': '/'
+}) }}
+{% endblock %}
+
+{% block content %}
+
+{{ govukPanel({
+'titleText': 'It appears you are already a member of the MoJ GitHub Organisation. Please sign in to GitHub and check
```
Comment (tamsinforbes) [inline app/templates/pages/already-a-member.html:18]: Shouldn't this state explicitly which org? `moj-analytical-services` or `ministryofjustice` or say 

```suggestion
'titleText': 'It appears you are already a member of the MoJ GitHub Organisation you attempted to join. Please sign in to GitHub and check
```

**Review by tamsinforbes — APPROVED** [review]

File: tests/routes/test_join.py [hunk 2]
```diff
@@ -130,6 +131,63 @@ def test_join_selection_other_user(self):
...
+            sess["org_selection"] = ["ministryofjustice"]
+
+        response = self.client.get("/join/send-invitation")
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.request.path, "/join/send-invitation")
+        self.assertIn(
+            "It appears you are already a member", str(
+                response.data))
+
+
```
Comment (connormaglynn) [inline tests/routes/test_join.py:190]: Could we move the tests from the Class below into this test Class? 👇 👀 

Just feels easier to read if all the tests for one function are under the same class 🤗 But happy for that to be a separate PR 😁

**Review by connormaglynn — APPROVED** [review]
🚀 Looks Good To Me! - with a small comment 🤏
